### ISSUE TITLE
Sectin 3.2 Discussion of backfill

### ISSUE THREAD

**Issue by wellsjc** [ATLAS-Titan/PanDA-WMS-paper#9]
FYI, the Adaptive Computing Manuel has a discussion of "backfill" that differs from ours in Section 3.2 in some respects. 
http://docs.adaptivecomputing.com/maui/8.2backfill.php
If necessary, we might consider adding a reference to the Adaptive Computing Manuel.

**Comment by mturilli** [comment]
I read the documentation and I agree we do not explain how backfill work but we just assume the reader knows about it. On the base of this assumption, we argue that enabling backfill can alleviate some of the shortcomings of not implementing pilots when submitting small/short jobs to Titan. Do you think this statement is reasonable/valid?

I added the indicated reference as a placeholder for an explanation of how backfill works. Do you think this is enough? 

Address by commit 5845a6f8d5b84993648bcdef836be6e823a8f82c

**Comment by wellsjc** [comment]
Matteo,

I am not sure I understand our backfill activity from the same perspective as you.  In my view, backfill on Titan works just fine until Titan's queue runs out of small, short jobs that have been submitted.   ATLAS provides, through the PanDA WMS a very large supply of small, short jobs to allow Moab to continue scheduling backfill jobs as opportunity allows.  PanDA increases the volume of small, short jobs that can appear in Titan's queue.

I really do not understand the phrase "enabling backfill can alleviate some of the shortcomings of not implementing pilots when submitting small/short jobs to Titan".

Jack

**Comment by wellsjc** [comment]
One approach may be to define a new term, for example, "marginal increase in backfill" or "marginal backfill increase".  My intent is to capture the core-hours used by PanDA/ATLAS that would have gone unused by Moab backfill as a result of insufficient, appropriate workload for instantaneous backfill.

**Comment by mturilli** [comment]
I think we may be on the same page, just from two different perspectives. Should we discuss this issue at our meeting today?

**Comment by wellsjc** [comment]
Yes, we should discuss, and this discussion will impact the data we show in Figure 3.
